### ISSUE TITLE
Project no longer supported?

### ISSUE THREAD

**Issue by imjosh** [docker/for-azure#74]
It seems that this project is no longer being supported.  Perhaps the readme should be updated to reflect this so people will stop using it for new projects and wondering if outstanding issues will ever be fixed?

**Comment by Praggie** [comment]
Yes, Please provide the status of this project. Should we stop using Docker on Azure?

**Comment by dealproc** [comment]
If this is the case, is the alternative to roll over to a k8s cluster in Azure?  I ask because I'm presently deployed on this within Azure, and have been considering the roll-over anyhow, so this would be the final nail in the coffin, sort-of speak.

**Comment by vesylapp** [comment]
@dealproc I ripped out docker4azure, and installed docker on ubuntu vms, configured the networking etc. and enabled swarm all manually.  For what I'm doing, the d4a template was overly complex anyway.   

I believe there are some solutions to automate some of that setup, or you could just create your own azure template if you know how to (I don't).  

I'm very familiar with swarm and it works very well for me; k8s has a steep learning curve, so I didn't want to have to go down that road.

**Comment by dealproc** [comment]
What intrigues me with k8s within azure is that you get the manager nodes gratis, whereas we're paying for the manager nodes with docker swarm using Docker-CE within azure.

**Comment by vesylapp** [comment]
The last time I checked (a while ago), ACS didn't seem fully baked to me.  Maybe it is by now.  I just let my managers also act as workers instead of paying for at least 3 VMs that aren't doing much.

**Comment by bitsofinfo** [comment]
Hey Docker employees or representatives: is Docker for Azure Dead? One simple answer YES or NO please?

**Comment by eosfor** [comment]
+1

**Comment by thehappycoder** [comment]
😭

**Comment by imjosh** [comment]
No reply from Docker in 3 months... 

![image](https://user-images.githubusercontent.com/3116028/50652104-64d69f00-0f53-11e9-8ad0-1ccbd385487d.png)

**Comment by dealproc** [comment]
Invest the time, learn k8s... it's worthwhile.
